wiki/Tournaments/DMX/3/en.md

```diff
@@ -85,37 +85,36 @@ The DeltaMax's Mania Key Smasher is run by various community members.
 
 -----------------------
 
-## Match Schedules: Semifinals
+## Match Schedules: Grand Finals
```

Comment:
maybe just add a note saying that the schedule is not released yet, so we can merge.